test/server/lib/recurring-contributions.test.js

```diff
@@ -442,7 +388,7 @@ describe('server/lib/recurring-contributions', () => {
         expect(entry.status).to.equal('failure');
 
         // And then the nextChargeDate is ajusted for two days later
-        expect(order.Subscription.nextChargeDate.getTime()).to.equal(new Date('2018-01-30 0:0').getTime());
+        expect(order.Subscription.nextChargeDate.getTime()).to.equal(moment().startOf('day').add(2, 'days').valueOf());
```

Comment:
I don't understand why we need to update this test, seems unrelated?

Reply:
I had to remove the `fakeTimer` because it was getting in conflict with `waitForCondition`.